### DISCUSSION TITLE
The ability to set chromium loclalStorage params

### DISCUSSION THREAD

**Discussion by nkiryanov** [gotenberg/gotenberg#890] [Q&A]
First of all, thanks for gotenberg — love it!

Question: Is it possible to set parameters in localStorage in chromium while capturing pdf/screenshots?

As far as I understand, currently there are only possibilities to set [cookies](https://gotenberg.dev/docs/routes#cookies-chromium) and [extraHttp](https://gotenberg.dev/docs/routes#custom-http-headers-chromium) headers, and setting a value in localStorage is not supported. Am i correct or did I miss something?

And if I understood everything correctly and there is no such functionality yet: would it be useful to make a PR for this?
It seems that chromedp [supports](https://pkg.go.dev/github.com/chromedp/cdproto/storage#SetSharedStorageEntry) that

**Comment by gulien** [comment]
Hey @nkiryanov,

If there is a use case for you, of course it would make sense. Also, you should add the ability to clear the local storage with a flag like `
--chromium-clear-cookies`.
